### ISSUE TITLE
The batch UI indicates a parsing error on some batches

### ISSUE THREAD

**Issue by robbecker-wf** [sourcegraph/sourcegraph#44189]
- **Sourcegraph version:** 4.0.1
- **Platform information:** Unsure, I think Workiva hosts our own instance internally

#### Steps to reproduce:

1. load the batch on our instance (https://sourcegraph.plat.workiva.net/organizations/Workiva/batch-changes/test_http_13)

#### Expected behavior:
I expect to see the list of PRs in the batch

#### Actual behavior:
I see `"Invalid character 'd' looking for beginning of value`
![image](https://user-images.githubusercontent.com/6053699/200983230-3a1d4e86-2e84-4bb2-a787-79f9fe2164d5.png)

If would like immediate help on this, please email support@sourcegraph.com (you can still create the issue, but there are not [SLAs](https://about.sourcegraph.com/support/) on issues like there are for support requests).

**Comment by BolajiOlajide** [comment]
Hello @robbecker-wf ,

I'm going to try to reproduce this on my machine. I know you specified in the platform information that Workiva hosts their own instance internally, but I'd like some information about the code hosts targeted via the batch spec.

If possible, I'll love to have the Batch Spec with all sensitive information taken out.

Thanks

**Comment by BolajiOlajide** [comment]
> Hello @robbecker-wf ,
> 
> I'm going to try to reproduce this on my machine. I know you specified in the platform information that Workiva hosts their own instance internally, but I'd like some information about the code hosts targeted via the batch spec.
> 
> If possible, I'll love to have the Batch Spec with all sensitive information taken out.
> 
> Thanks

Hey! I found the root cause of this issue. (Andrew will also be reaching out regarding this via Slack). 
The bug report above was caused by a change in the database schema that was introduced in v4.0, this was fixed in [4.1.1](https://github.com/sourcegraph/sourcegraph/pull/43390). 
The bug is caused by draft changesets, we changed the `published` column type to work with certain types.
The two solutions that we can explore are:

1. Upgrade your instance to 4.1.1
2. Manually update the changesets in the DB (just a one-time update) with this query:

```sql
UPDATE changeset_specs
SET published = '"draft"'
WHERE published = 'draft';
```

**Comment by robbecker-wf** [comment]
We're upgraded to 4.1.3 and I haven't seen the issue anymore. I'm going to close this issue. Thanks!
